engine-rest/engine-rest-openapi/src/main/templates/paths/task/{id}/localVariables/{varName}/put.ftl

```diff
@@ -0,0 +1,63 @@
+{
+
+  <@lib.endpointInfo
+      id = "putTaskLocalVariable"
+      tag = "Task Local Variable"
+      desc = "Sets a variable in the context of a given task." />
+
+  "parameters": [
+
+    <@lib.parameter
+        name = "id"
+        location = "path"
+        type = "string"
+        required = true
+        desc = "The id of the task to set the variable for." />
+
+    <@lib.parameter
+        name = "varName"
+        location = "path"
+        type = "string"
+        required = true
+        last = true
+        desc = "The name of the variable to set." />
+
+  ],
+
+  <@lib.requestBody
+      mediaType = "application/json"
+      dto = "VariableValueDto"
+      examples = [ '"example-1": {
+                       "summary": "PUT /task/aTaskId/variables/aVarName",
+                       "description": "Status 204. No content.",
+                       "value": {
+                         "value": "someValue",
+                         "type": "String"
+                       }
+                     }'
+      ] />
```

Comment:
The second example is missing. I think we should include it or adjust the docs as both examples are pretty similar. I would choose example 2 over the one used here, because it shows more parameters.

Reply:
I would add both since the first shows a primitive type variable, and the second, a more complex Object var. It's good to show the differences in the request.